db/migrate/20160610124506_add_lives_on_to_middleware_server.rb

```diff
@@ -0,0 +1,11 @@
+class AddLivesOnToMiddlewareServer < ActiveRecord::Migration[5.0]
```

Comment:
typically yes, but what's not reversible in `add_column` operation? Rails "knows" that its "reverse" is remove column hence you can use here `change`.
When should you prefer to use up/down rather than `change`? For example, if during `up` when you add a column you'd also want to populate all rows with some default value. Then change would be more tricky and less clear.